exporter/prometheus/prometheus.go

```diff
@@ -163,6 +162,10 @@ type collector struct {
 	registeredViewsMu sync.Mutex
 	// registeredViews maps a view to a prometheus desc.
 	registeredViews map[string]*prometheus.Desc
+
+	// unregisteredViews is the set of views that are
```

Comment:
s/unregisteredViews/newViews

Unregistered sounds like they have been registered and unregistered at some point.